week1/05.py

```diff
@@ -0,0 +1,4 @@
+weight = float(input("Please input your weight in kilogram: "))
+height = float(input("Please input your height in meter: "))
+
+print("Your body mass index is: ", round(weight / (height * height), 2))
```

Comment:
`height * height` can be written more elegantly in python, try to find the power to operator